### ISSUE TITLE
Unmet peer dependency "prop-types@^15.5.10".

### ISSUE THREAD

**Issue by tashburn** [FortAwesome/react-fontawesome#103]
When I do a `yarn` operation like `add` or `remove`, I get this warning --

```
warning " > @fortawesome/react-fontawesome@0.0.17" has unmet peer dependency "prop-types@^15.5.10".
```

I'm using the latest version of `prop-types`, which is `15.6.1`. Please update to avoid this warning.

I have "@fortawesome/react-fontawesome": "0.0.17",

**Comment by robmadole** [comment]
Should be fixed with 0.0.18. Let me know if there are still problems.

**Comment by tashburn** [comment]
@robmadole  Thank you
